lapack/lapack64/lapack64.go

```diff
@@ -420,6 +420,21 @@ func Ggsvd3(jobU, jobV, jobQ lapack.GSVDJob, a, b blas64.General, alpha, beta []
 	return lapack64.Dggsvd3(jobU, jobV, jobQ, a.Rows, a.Cols, b.Rows, a.Data, max(1, a.Stride), b.Data, max(1, b.Stride), alpha, beta, u.Data, max(1, u.Stride), v.Data, max(1, v.Stride), q.Data, max(1, q.Stride), work, lwork, iwork)
 }
 
+// Gtsv solves the equation
+//  A * X = B
+// where A is an n×n tridiagonal matrix. It uses Gaussian elimination with
+// partial pivoting.
+//
+// On entry, a contains the matrix A, on return it will be overwritten.
+//
+// On entry, b contains the n×nrhs right-hand side matrix B. On return, if ok is
+// true, it will be overwritten by the solution matrix X.
+//
+// Gtsv returns whether the solution X has been successfuly computed.
+func Gtsv(a Tridiagonal, b blas64.General) (ok bool) {
```

Comment:
Having a `trans` parameter here would actually be useful. Are you fine with adding it, @kortschak ?

Reply:
In an isolated type `.T() Tridiagonal` would be best but as you write it's different to all other blas64 types, so I've added a trans parameter here. I think we have some freedom in deciding lapack64 package API.